sentry-android-core/src/main/java/io/sentry/android/core/performance/ActivityLifecycleTimeSpan.java

```diff
@@ -0,0 +1,15 @@
+package io.sentry.android.core.performance;
+
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+@ApiStatus.Internal
+public class ActivityLifecycleTimeSpan implements Comparable<ActivityLifecycleTimeSpan> {
+  public final @NotNull TimeSpan onCreate = new TimeSpan();
```

Comment:
i know it's "verbose", but Java likes getters 😅 
make it private with a public getter